frontend/src/pages/admin/Template/index.vue

```diff
@@ -124,6 +124,20 @@ export default {
                             modulesChanged = modulesChanged || pmChanges.changed
                             changed = changed || pmChanges.changed
                             errors = errors || pmChanges.errors
+                        } else if (field === 'palette_catalogue') {
+                            this.editable.changed.settings.palette_catalogue = false
+                            if (this.original.settings.palette_catalogue.length !== this.editable.settings.palette_catalogue.length) {
+                                this.editable.changed.settings.palette_catalogue = true
+                            } else {
+                                for (const i in this.editable.settings.palette_catalogue) {
+                                    const d = this.editable.settings.palette_catalogue[i] !== this.original.settings.palette_catalogue[i]
```

Comment:
What is `d`?

Reply:
It stood for delta, but we could just move the comparison into the if the line below and remove it